shell_scripts/idm_postprocessing.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#SBATCH --output=sbatch_logs/postprocess_results/postprocess_results_%A.log
+#SBATCH --job-name=postprocess_results
+#SBATCH --time=30-00:00
+#SBATCH --mem=8G
+#SBATCH -c 2
+#SBATCH --nodelist=idmodeling2
+
+echo "***** Beginning of script *****"
+date
+
+TAXDIR=/home/cholera_mapping_pipeline 
```

Comment:
- The user name will need to be added manually to the path, no? In the other shell scripts we had /home/.../cholera-mapping-pipeline 
- Also - should this be underscore or dashes between cholera mapping pipeline? Even if your folder has underscores, I think dashes should be the default since that is the repo name

Reply:
sounds good, I'll change that with user name and use the default folder name.